content/docs/5.1/deploy/benchmark/benchmark-instructions.md

```diff
@@ -0,0 +1,155 @@
+1. Load the database, using workload A's parameter file (workloads/workloada) and the "-load" switch to the client.
+2. Run workload A (using workloads/workloada and "-t") for a variety of throughputs.
+3. Run workload B (using workloads/workloadb and "-t") for a variety of throughputs.
+4. Run workload C (using workloads/workloadc and "-t") for a variety of throughputs.
+5. Run workload F (using workloads/workloadf and "-t") for a variety of throughputs.
+6. Run workload D (using workloads/workloadd and "-t") for a variety of throughputs. This workload inserts records, increasing the size of the database.
+7. Delete the data in the database.
+8. Reload the database, using workload E's parameter file (workloads/workloade) and the "-load switch to the client.
+9. Run workload E (using workloads/workloade and "-t") for a variety of throughputs. This workload inserts records, increasing the size of the database.
+
+{{< info >}}
+If you trying to use more clients to benchmark, check [Running a Workload in Parallel](https://github.com/brianfrankcooper/YCSB/wiki/Running-a-Workload-in-Parallel).
+{{< /info >}}
+
+For example, you can load a workload contains 10M records and 30M operations by:
+
+```sh
+go-ycsb load tikv -P workloads/workloada -p tikv.pd="10.0.1.1:2379" -p tikv.type="raw" -p recordcount=10000000 -p operationcount=30000000
+```
+
+After successfully loaded the data, you can launch it.
+
+```sh
+go-ycsb run tikv -P workloads/workloada -p tikv.pd="10.0.1.1:2379" -p tikv.type="raw" -p recordcount=10000000 -p operationcount=30000000
+```
+
+You can specify the concurrency of the workload client by `-p threadcount=`. Normally, the number should be the number of virtual CPU cores binded to the machine.
+
+## Step 4. Inspect the benchmark results
+
+While running `go-ycsb`, it will reports workload runtime information like the OPS, latency:
+
+```
+...
+READ   - Takes(s): 9.7, Count: 110092, OPS: 11380.1, Avg(us): 3822, Min(us): 236, Max(us): 342821, 99th(us): 51000, 99.9th(us): 59000, 99.99th(us): 339000
+UPDATE - Takes(s): 9.7, Count: 110353, OPS: 11408.8, Avg(us): 7760, Min(us): 944, Max(us): 344934, 99th(us): 59000, 99.9th(us): 65000, 99.99th(us): 339000
+READ   - Takes(s): 19.7, Count: 229147, OPS: 11647.2, Avg(us): 3094, Min(us): 202, Max(us): 342821, 99th(us): 52000, 99.9th(us): 58000, 99.99th(us): 335000
+```
+
+It will report the summary of the workload whlie the workload is done.
+
+```
+Run finished, takes 4m25.292657267s
+READ   - Takes(s): 265.0, Count: 4998359, OPS: 18864.7, Avg(us): 1340, Min(us): 181, Max(us): 342821, 99th(us): 11000, 99.9th(us): 51000, 99.99th(us): 73000
+UPDATE - Takes(s): 265.0, Count: 5001641, OPS: 18877.1, Avg(us): 5416, Min(us): 599, Max(us): 1231403, 99th(us): 53000, 99.9th(us): 276000, 99.99th(us): 772000
+```
+
+{{< warning >}}
+If it report errors like `batch commands send error:EOF`, it relates to this [issue](https://github.com/pingcap/go-ycsb/issues/145).
+{{< /warning >}}
+
+
+## Step 5. Clean up
```

Comment:
How can users analysis the benchmark result？e.g. how to find the bottleneck？

Reply:
What does `bottleneck` means? Is the maximum throughput of `TiKV` or the components that are not fully used.